### PR TITLE
docs: description of disabled button

### PR DESCRIPTION
**Pull request by annehubtype**
The following files have been updated based on the new button disabling feature:
- components-button
- webchat-style
- sidebars 
The following file has been created:
- webchat-button-disable (with new images)

+ Also removed typos in humanhandoff file

## Testing

The pull request...

- has unit tests
- has integration tests
- doesn't need tests because... **[provide a description]**

### CODE REVIEW

File: docs/docs/components/components-button.md [hunk 1]
```diff
@@ -16,9 +16,10 @@ The `Button` component is used to display a button during a conversation. The us
 | path     | String         |                                                                    | No       | -             |
 | children | String         | Show the button                                                    | Yes      | -             |
 | payload  | String, Number | Call to action                                                     | No       | -             |
-| url      | String         | This URL is opened in a mobile browser when the button is clicked. | No       | -             |
+| url      | String         | This URL is opened in a mobile browser when the button is clicked | No       | -             |
 | onClick  | String         | Set the action                                                     | No       | -             |
 | target  | String         | Opens the linked page in a new window **(_blank)** or same frame **(_self)** or parent frame **(_parent)** or full body of the window **(_top)** or **framename**                                                    | No       | _blank         |
+| autodisable  | Boolean         | Disable a button once the user has clicked on it        | No       | true          |
```
Comment (vanbasten17) [inline docs/docs/components/components-button.md:22]: missing `disabledstyle` here. Appearance of the button once it has been disabled

File: docs/docs/webchat/webchat-button-disable.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
+---
+id: webchat-button-disable
+title: Enabling and Disabling Buttons
+---
+
+This functionality allows you to disable a button once the user has clicked on it. Thanks to this visual information, the user can better understand the logical flow and navigate more easily.
```
Comment (vanbasten17) [inline docs/docs/webchat/webchat-button-disable.md:6]: Also to force the user to follow an specific flow. (as he won't be able to click that button again)

File: docs/docs/webchat/webchat-button-disable.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+To add this property by default, you must set `autodisable: true` in the webchat `index.js`.
+The button style properties must be set as well.
```
Comment (vanbasten17) [inline docs/docs/webchat/webchat-button-disable.md:9]: The button style properties must be set as well, is wrong, we provide a default style of a disabled button, but they can be customized with disabledstyle

File: docs/docs/webchat/webchat-button-disable.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+      autodisable: true,
+    },
+  },
+}
+```
+
+
+## Example with a button
+
+In `button.js`, if you add:
```
Comment (vanbasten17) [inline docs/docs/webchat/webchat-button-disable.md:35]: You can also overwrite this behavior in a more specific level by defining these props in a specific `Button`.

File: docs/docs/webchat/webchat-button-disable.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+          ))}
+        </Carousel>
+```
+
+You should get something like:
+
+<img src="https://botonic-doc-static.netlify.com/images/webchat/carousel-enable.png" width="400" />
+
+
+<img src="https://botonic-doc-static.netlify.com/images/webchat/carousel-disable.png" width="400" />
```
Comment (vanbasten17) [inline docs/docs/webchat/webchat-button-disable.md:87]: I would not put these images as examples as they are not intended to be used directly in the docs but to facilitate your understanding when extracting the information from the PR. You can just leave the images in Buttons which are more "serious" and just say that the same behavior can be achieved in carousel buttons

File: docs/docs/webchat/webchat-style.md [hunk 6]
```diff
@@ -91,7 +91,8 @@ Here is a list of available styles. You need to modify them inside **src/webchat
...
 | `blockInputs`           | The inputs not allowed by the bot. It can also be defined in `userInput.blockInputs`.                                                                                                         |
 | `persistentMenu`        | An array containing the options of your persistent menu. It can also be defined in `userInput.persistentMenu`.                                                                                |
-| `enableAttachments`     | Whether to allow user media attachments or not (`true                                                                                                                                         | false`). It can also be defined in `userInput.attachments.enable`. `false` by default. |
+| `enableAttachments`     | Whether to allow user media attachments or not (`true | false`). It can also be defined in `userInput.attachments.enable`. `false` by default. |
+ `disabledstyle`     | Style of a disabled button once clicked. To be defined only if the `autodisable` property is set to `true`. |
```
Comment (vanbasten17) [inline docs/docs/webchat/webchat-style.md:95]: not here, but in the other table.

**Review by vanbasten17 — CHANGES_REQUESTED** [review]

File: docs/docs/webchat/webchat-button-disable.md (hunk 5 above)
Comment (annehubtype) [reply docs/docs/webchat/webchat-button-disable.md]: Example and images removed.

**Review by vanbasten17 — APPROVED** [review]
In `Custom Styles` (https://botonic.io/docs/webchat/webchat-style#custom-styles). You should add:
* `documentDownload`
* buttonDisabledStyle: (explanation + `It can also be defined in button.disabledstyle`)
* buttonAutoDisable: (explanation + `It can also be defined in button.autodisable`)

File: docs/docs/webchat/webchat-style.md [hunk 7]
```diff
@@ -91,7 +93,8 @@ Here is a list of available styles. You need to modify them inside **src/webchat
...
 | `blockInputs`           | The inputs not allowed by the bot. It can also be defined in `userInput.blockInputs`.                                                                                                         |
 | `persistentMenu`        | An array containing the options of your persistent menu. It can also be defined in `userInput.persistentMenu`.                                                                                |
-| `enableAttachments`     | Whether to allow user media attachments or not (`true                                                                                                                                         | false`). It can also be defined in `userInput.attachments.enable`. `false` by default. |
+| `enableAttachments`     | Whether to allow user media attachments or not (`true | false`). It can also be defined in `userInput.attachments.enable`. `false` by default. |
+| `documentDownload`      | Styles of the button displayed to download a document sent via the webchat.   |
```
[the remark this answers is not in the document]
Comment (annehubtype) [reply docs/docs/webchat/webchat-style.md:97]: @vanbasten17 It's already in Custom Styles section (in the table)

[the remark this answers is not in the document]
Comment (vanbasten17) [reply docs/docs/webchat/webchat-style.md]: nope, right now it is in the table below `Custom Webchat Example`, it has to be in below table.

File: docs/docs/webchat/webchat-style.md [hunk 8]
```diff
@@ -24,6 +24,7 @@ export const webchat = {
   enableUserInput: {true|false},
   enableAttachments: {true|false},
   enableAnimations: {true|false},
+  autodisable: {true|false},
```
Comment (vanbasten17) [inline docs/docs/webchat/webchat-style.md:27]: remove this

**Review by vanbasten17 — APPROVED** [review]
Now it's okay! 👍

File: docs/docs/components/components-button.md [hunk 9]
```diff
@@ -16,9 +16,11 @@ The `Button` component is used to display a button during a conversation. The us
...
```
Comment (AinaVendrell) [inline docs/docs/components/components-button.md:22]: Shouldn't the default value of autodisable be `false` @marcrabat?

File: docs/docs/webchat/webchat-style.md [hunk 10]
```diff
@@ -76,6 +76,8 @@ Here is a list of available styles. You need to modify them inside **src/webchat
 | `buttonStyle`           | Button styles. It can also be defined in `button.style`.                                                                                                                                      |
 | `buttonHoverBackground` | Background color when hovering over a button. It can also be defined in `button.hoverBackground`.                                                                                             |
 | `buttonHoverTextColor`  | Text color when hovering over a button. It can also be defined in `button.hoverTextColor`.                                                                                                    |
+| `buttonAutoDisable`    | Disables a button once the user has clicked on it. Default value:`true`. It can also be defined in `button.autodisable`. |
```
Comment (AinaVendrell) [inline docs/docs/webchat/webchat-style.md:79]: same here, default value false?

**Review by AinaVendrell — APPROVED** [review]

File: docs/docs/components/components-button.md (hunk 9 above)
Comment (annehubtype) [reply docs/docs/components/components-button.md:22]: Shouldn't the default value of autodisable be false @vanbasten17

File: docs/docs/webchat/webchat-style.md (hunk 10 above)
Comment (annehubtype) [reply docs/docs/webchat/webchat-style.md:79]: same here, default value false? @vanbasten17

Comment (vanbasten17) [reply docs/docs/webchat/webchat-style.md:79]: yes, the default value is false

File: docs/docs/components/components-button.md (hunk 9 above)
Comment (vanbasten17) [reply docs/docs/components/components-button.md:22]: false by default

**Review by ericmarcos — APPROVED** [review]

**Pull request merged**